### PR TITLE
Fix chrome version

### PR DESCRIPTION
**Pull request by Infobeans-Anujita**
The conftest.py file contains all the changes which parameterizes all the arguments from command line.

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -1,32 +1,67 @@
...
-    desired_caps['platformName'] = 'Linux'
-    desired_caps['platformVersion'] = '51.0.2704.106'
-    desired_caps['browserName'] = browser
-    driver = webdriver.Remote('http://localhost:4444/wd/hub', desired_caps)
-    return driver
+def setUpWeb(browser,mode,platformName,platformVersion,hub_url):
+    if mode == 'local':
+        if browser == 'chrome':
+            driver = webdriver.Chrome()
+            driver.maximize_window()
```
Comment (ersuniljoshi) [inline conftest.py]: remove redundant code

File: conftest.py [hunk 2]
```diff
@@ -1,32 +1,67 @@
...
+    parser.addoption("--platformName", action="store", default="Linux", help="name: Linux")
+    parser.addoption("--platformVersion", action="store", default="55.0", help="version: 55.0")
+    parser.addoption("--hub_url", action="store", default="http://localhost:4444/wd/hub",
+                     help="url: http://localhost:4444/wd/hub")
 
 
 @pytest.fixture
 def codexFile(request):
-    return request.config.getoption("--codexFile")
+    return request.config.getoption("--codexFile").lower()
```
Comment (ersuniljoshi) [inline conftest.py]: Think again and use if required.

File: conftest.py [hunk 3]
```diff
@@ -1,32 +1,67 @@
 # Using appium webdriver to automate
```
Comment (ersuniljoshi) [inline conftest.py]: ReadMe not updated, please send it.

File: conftest.py [hunk 4]
```diff
@@ -1,32 +1,67 @@
...
-from appium import webdriver
+from selenium import webdriver
 import pytest
 import os
 
 
 def pytest_addoption(parser):
     parser.addoption("--codexFile", action="store", default="web", help="my option: web")
```
Comment (ersuniljoshi) [inline conftest.py]: Please run pep8 and flake8 on your code.

File: conftest.py [hunk 5]
```diff
@@ -1,32 +1,67 @@
...
+def mode(request):
+    return request.config.getoption("--mode").lower()
+
+@pytest.fixture
+def platformName(request):
+    return request.config.getoption("--platformName").lower()
+
+@pytest.fixture
+def platformVersion(request):
+    return request.config.getoption("--platformVersion")
```
Comment (ersuniljoshi) [inline conftest.py]: Add lower if needed

File: conftest.py (hunk 2 above)
Comment (Infobeans-Anujita) [inline conftest.py]: Removed converting to lowercase because it may create some issues when extracting codex file

File: conftest.py (hunk 5 above)
Comment (Infobeans-Anujita) [inline conftest.py]: Done

File: conftest.py (hunk 1 above)
Comment (Infobeans-Anujita) [inline conftest.py]: Done

File: conftest.py (hunk 4 above)
Comment (Infobeans-Anujita) [inline conftest.py]: Done

**Pull request merged**